src/shopilingus/ShoppingCenter.java

```diff
@@ -0,0 +1,174 @@
+package shopilingus;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ShoppingCenter {
+
+    private static int nextId = 0;
+
+    private int id;
+    private String name;
+    private String address;
+    private List<Shop> shops;
+
+    public ShoppingCenter(String name, String address) {
+        this.id = ShoppingCenter.nextId++;
+        this.name = name;
+        this.address = address;
+        this.shops = shops;
+    }
+
+    public static int getNextId() {
+        return nextId;
+    }
+
+    public static void setNextId(int nextId) {
+        ShoppingCenter.nextId = nextId;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getAddress() {
+        return address;
+    }
+
+    public void setAddress(String address) {
+        this.address = address;
+    }
+
+    public List<Shop> getShops() {
+        return shops;
+    }
+
+    public void setShops(List<Shop> shops) {
+        this.shops = shops;
+    }
+
+
+
+    public void addShop(Shop shop) {
+
+        this.shops.add(shop);
+
+    }
+
+    public void deleteShop(int shopId) {
+
```

Comment:
let's assume that `delete` removes if object exists, if not found, let's ignore it for delete (no need to print message)